railties/lib/rails/generators/rails/app/templates/test/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
+    take_failed_screenshot
+    Capybara.reset_sessions!
```

Comment:
They can't be in the base class because if someone adds something to teardown then reset_sessions will not be last and will not work correctly.